src/server/scripts/World/go_scripts.cpp

```diff
@@ -1160,7 +1160,7 @@ class go_brewfest_music : public GameObjectScript
                     if (!IsHolidayActive(HOLIDAY_BREWFEST)) // Check if Brewfest is active
                         break;
                     rnd = urand(0, 2); // Select random music sample
-                    _events.ScheduleEvent(EVENT_BM_SELECT_MUSIC, musicTime); // Select new song music after play time is over
+                    _events.ScheduleEvent(EVENT_BM_SELECT_MUSIC, Milliseconds(musicTime)); // Select new song music after play time is over
```

Comment:
What stopped you from turning musicTime into milliseconds on class member level?

Reply:
That was before we discussed the constexpr constants. In this case I really didn't want to remove the named values. Going to change it.

Reply:
Done.